test_opensearch/test_server_secured/test_clients.py

```diff
@@ -0,0 +1,48 @@
+# -*- coding: utf-8 -*-
+# SPDX-License-Identifier: Apache-2.0
+#
+# The OpenSearch Contributors require contributions made to
+# this file be licensed under the Apache-2.0 license or a
+# compatible open source license.
+#
+# Modifications Copyright OpenSearch Contributors. See
+# GitHub history for details.
+#
+#  Licensed to Elasticsearch B.V. under one or more contributor
+#  license agreements. See the NOTICE file distributed with
+#  this work for additional information regarding copyright
+#  ownership. Elasticsearch B.V. licenses this file to you under
+#  the Apache License, Version 2.0 (the "License"); you may
+#  not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+# 	http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing,
+#  software distributed under the License is distributed on an
+#  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+#  KIND, either express or implied.  See the License for the
+#  specific language governing permissions and limitations
+#  under the License.
+
+from unittest import TestCase
+
+from opensearch import OpenSearch
+from opensearch.helpers.test import OPENSEARCH_URL
+
+
+class TestSecurity(TestCase):
+    def test_security(self):
+        client = OpenSearch(
+            OPENSEARCH_URL,
+            http_auth=("admin", "admin"),
+            verify_certs=False,
+        )
+
+        info = client.info()
+        self.assertNotEqual(info["version"]["number"], "")
+        self.assertNotEqual(info["tagline"], "")
+        self.assertTrue(
+            info["version"].get("build_flavor", "") != ""
+            or info["version"].get("distribution", "") != ""
+        )
```

Comment:
Have no idea what OpenSearch build with different options can return, but does that mean that keys inside `version` can be an empty string which is a criterion of failure? (Trying to understand, is there any motivation to not use e.g. `"build_flavor" in info["version"]` syntax or not).

Reply:
i agree, ```"build_flavor" in info["version"]``` is more pythonic than using !=

Reply:
Addressed in latest commit